### PR TITLE
init actor

### PR DESCRIPTION
**Pull request by vuhust19081995**

### CODE REVIEW

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -20,4 +20,6 @@
     <dimen name="poster_image_width">130dp</dimen>
     <dimen name="text_overview_size">13sp</dimen>
     <dimen name="dp_3">3dp</dimen>
+    <dimen name="dp_100">100dp</dimen>
```
Comment (DoanVanToan) [inline app/src/main/res/values/dimens.xml]: WARNING: [UnusedResources] The resource `R.dimen.dp_100` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -20,4 +20,6 @@
...
+    <dimen name="dp_70">70dp</dimen>
```
Comment (DoanVanToan) [inline app/src/main/res/values/dimens.xml]: WARNING: [UnusedResources] The resource `R.dimen.dp_70` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/dimens.xml (hunk 1 above)
Comment (vuhust19081995) [reply app/src/main/res/values/dimens.xml]: usedResources

File: app/src/main/res/values/dimens.xml (hunk 2 above)
Comment (vuhust19081995) [reply app/src/main/res/values/dimens.xml]: usedResources

**Pull request merged**